e2e-tests/commands/insurance/check-policy-summary-list.js

```diff
@@ -109,15 +109,6 @@ const checkPolicySummaryList = ({
 
       const expectedChangeLinkText = FIELDS.CONTRACT_POLICY.SINGLE[fieldId].SUMMARY.TITLE;
 
-      cy.assertSummaryListRow(summaryList, fieldId, expectedKey, expectedValue, expectedChangeLinkText);
-    },
-    [TOTAL_CONTRACT_VALUE]: () => {
-      const fieldId = TOTAL_CONTRACT_VALUE;
-      const expectedKey = FIELDS.CONTRACT_POLICY.SINGLE[fieldId].SUMMARY.TITLE;
-      const expectedValue = formatCurrency(application.POLICY[fieldId]);
-
-      const expectedChangeLinkText = FIELDS.CONTRACT_POLICY.SINGLE[fieldId].SUMMARY.TITLE;
-
```

Comment:
Should this be updated instead of removed?

Reply:
not yet as not data saving.  Will be readded on next PR

Reply:
readded